rocketpy/prints/sensors_prints.py

```diff
@@ -0,0 +1,98 @@
+from abc import ABC, abstractmethod
+
+UNITS = {
+    "Gyroscope": "rad/s",
+    "Accelerometer": "m/s^2",
+    "Magnetometer": "T",
+    "PressureSensor": "Pa",
+    "TemperatureSensor": "K",
+}
+
+
+class _SensorsPrints(ABC):
+    def __init__(self, sensor):
+        self.sensor = sensor
+        self.units = UNITS[sensor.type]
+
+    def _print_aligned(self, label, value):
```

Comment:
I call this in the child classes, so it cant be double underscore

Reply:
Question: is the output of sensor prints really convinient?

Take this example:
```text
Identification of the Sensor:

Name:                      Accelerometer in Nosecone
Type:                      Accelerometer

Orientation of the Sensor:

Orientation:               (60, 60, 60)
Normal Vector:             (0.9665063509461147, -0.058012701892006885, 0.2500000000000297)
Rotation Matrix:
                           [0.25, -0.06, 0.97]
                           [0.43, 0.9, -0.06]
                           [-0.87, 0.43, 0.25]

Quantization of the Sensor:

Measurement Range:         -70 to 70 (m/s^2)
Resolution:                0.4882 m/s^2/LSB

Noise of the Sensor:

Noise Density:             (0.05, 0.05, 0.05) m/s^2/√Hz
Noise Variance:            (1, 1, 1) (m/s^2)^2
Random Walk Density:       (0.02, 0.02, 0.02) m/s^2/√Hz
Random Walk Variance:      (1, 1, 1) (m/s^2)^2
Constant Bias:             (1, 1, 1) m/s^2
Operating Temperature:     25 °C
Temperature Bias:          (0.02, 0.02, 0.02) m/s^2/°C
Temperature Scale Factor:  (0.02, 0.02, 0.02) %/°C
Cross Axis Sensitivity:    0.02 %
Identification of the Sensor:

Name:                      Accelerometer in CDM
Type:                      Accelerometer

Orientation of the Sensor:

Orientation:               [[0.25, -0.0581, 0.9665], [0.433, 0.8995, -0.0581], [-0.8661, 0.433, 0.25]]
Normal Vector:             (0.9665010341566599, -0.05810006216709978, 0.25000026750042936)
Rotation Matrix:
                           [0.25, -0.06, 0.97]
                           [0.43, 0.9, -0.06]
                           [-0.87, 0.43, 0.25]

Quantization of the Sensor:

Measurement Range:         -inf to inf (m/s^2)
Resolution:                0 m/s^2/LSB

Noise of the Sensor:

Noise Density:             (0, 0, 0) m/s^2/√Hz
Noise Variance:            (1, 1, 1) (m/s^2)^2
Random Walk Density:       (0, 0, 0) m/s^2/√Hz
Random Walk Variance:      (1, 1, 1) (m/s^2)^2
Constant Bias:             (0, 0, 0) m/s^2
Operating Temperature:     25 °C
Temperature Bias:          (0, 0, 0) m/s^2/°C
Temperature Scale Factor:  (0, 0, 0) %/°C
Cross Axis Sensitivity:    0 %

```

Some suggestions:
* Use "----------------" to separate sections and sub-sections
* It is annoying to have to read "... of the sensor" in every section. Remove it.

Reply:
https://github.com/RocketPy-Team/RocketPy/pull/583/commits/82f79e7433d97f09ea6c9acb54d0ef72dfb030c7

I didn't add the lines because they look cumbersome in this case